### DISCUSSION TITLE
Using external storage services for images and files

### DISCUSSION THREAD

**Discussion by sathyarajv** [danny-avila/LibreChat#1865] [Feature Requests & Suggestions]
Looking for a config support to move file and image storage to external storage services like S3, Azure Storage service, etc.,

**Comment by fuegovic** [comment]
Firebase CDN is currently supported:
https://docs.librechat.ai/features/firebase.html

**Comment by ndrsfel** [comment]
Depending on your hosting setup (I assume docker) you can use local storage and map that path to persistent storage. That’s how I do it in Azure using blob storage.

**Reply by oza-c** [reply]
Which resource do you use for hosting? Azure WebApp or Azure Container Instances/App?
